example/index.js

```diff
@@ -17,35 +17,44 @@ function getReplaceStr() {
 }
 
 function getResponsive() {
-  return document.querySelector('input.responsive-input').checked;
+  return document.querySelector('input#responsive-input').checked;
 }
 
-function oneRow() {
+function ellipsize(rows) {
   reset();
-  ellipsis('.text p.aaa', 1, { replaceStr: getReplaceStr(), responsive: getResponsive() });
-  ellipsis('.text p.lorem-ipsum', 1, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsis('.text p.aaa', rows, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsis('.text p.lorem-ipsum', rows, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  didEllipsize = true;
+}
+
+function oneRow() {
+  ellipsize(1);
 }
+document.getElementById('one-row').addEventListener('click', oneRow);
 
 function twoRows() {
-  reset();
-  ellipsis('.text p.aaa', 2, { replaceStr: getReplaceStr(), responsive: getResponsive() });
-  ellipsis('.text p.lorem-ipsum', 2, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsize(2);
 }
+document.getElementById('two-rows').addEventListener('click', twoRows);
 
 function threeRows() {
-  reset();
-  ellipsis('.text p.aaa', 3, { replaceStr: getReplaceStr(), responsive: getResponsive() });
-  ellipsis('.text p.lorem-ipsum', 3, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsize(3);
 }
+document.getElementById('three-rows').addEventListener('click', threeRows);
 
 function fourRows() {
-  reset();
-  ellipsis('.text p.aaa', 4, { replaceStr: getReplaceStr(), responsive: getResponsive() });
-  ellipsis('.text p.lorem-ipsum', 4, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsize(4);
 }
+document.getElementById('four-rows').addEventListener('click', fourRows);
 
 function fiveRows() {
-  reset();
-  ellipsis('.text p.aaa', 5, { replaceStr: getReplaceStr(), responsive: getResponsive() });
-  ellipsis('.text p.lorem-ipsum', 5, { replaceStr: getReplaceStr(), responsive: getResponsive() });
+  ellipsize(5);
+}
+document.getElementById('five-rows').addEventListener('click', fiveRows);
+
+var didEllipsize = false;
+
+function responsiveListener() {
+  didEllipsize && window.location.reload();
```

Comment:
It's probably the only way (I'm referring to discussion about this in #27), but it's - you know - a bit violent for the user.
I'd add it only if the user has already called `ellipsis` with the responsive option flagged, because in the other case he will select `responsive` and the page will reload leaving `responsive` unflagged as default.

Do you understand what I mean?

Reply:
Huh, yeah, my mistake... I actually did that in the first place, but I was experimenting a more difficult approach and had to clean up. I think I removed that by mistake, gonna re-add it right now.

Reply:
Fixed this and added the commit

Reply:
Exactly, it's ok now